batch/batch/batch.py

```diff
@@ -279,28 +288,28 @@ def _keep_alive_container(self):  # pylint: disable=R0201
         _, err = await app['k8s'].create_pod(body=pod_template)
         if err is not None:
             if err.status == 409:
-                log.info(f'pod already exists for job {self.id}')
+                self.log_info(f'pod already exists')
                 return
             traceback.print_tb(err.__traceback__)
-            log.info(f'pod creation failed for job {self.id} '
-                     f'with the following error: {err}')
+            self.log_info(
+                f'pod creation failed with the following error: {err}')
```

Comment:
This can be one line now

Reply:
it's 81 characters, but I made it one line ;)